tools/sdk/resourcemanager/terraform.go

```diff
@@ -232,6 +232,9 @@ type ResourceIdMappingDefinition struct {
 	// SegmentName is the name of the ResourceId Segment which should be mapped to/from
 	// the Schema Field defined in SchemaFieldName.
 	SegmentName string `json:"segmentName"`
+
+	// ParsedFromParentID specifies whether the field in SchemaFieldName is a parent resource.
+	ParsedFromParentID bool `json:"parent"`
```

Comment:
we'll also want to change the json field name (as above):

```suggestion
	ParsedFromParentID bool `json:"parsedFromParentId"`
```